### PR TITLE
Refactor/websocket

### PR DESCRIPTION
**Pull request by Bogh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ cover.out
 .goxc.json
 /.idea
 *.iml
+app
```
Comment (marian-craciunescu) [inline .gitignore]: please remove this

File: server/websocket_connector_test.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ var aTestMessage = &guble.Message{
 	Body: []byte("Test"),
 }
 
-func Test_WSHandler_SubscribeAndUnsubscribe(t *testing.T) {
+func TestWebSocket_SubscribeAndUnsubscribe(t *testing.T) {
```
Comment (smancke) [inline server/websocket_connector_test.go]: So far the convention was: Test_<Type>_<Description>
Changing to Test<Type>_<Description> would be fine for me, but than should be done consistently.

Comment (smancke) [inline server/websocket_connector_test.go]: So far, the convention was Test_<Type>_<Description>
Changing to Test<Type>_<Description> is ok for me if you prefer it, but should be done consistently in all test.

Comment (Bogh) [inline server/websocket_connector_test.go]: True. We agreed to use the underscore version.

**Comment by smancke** [comment]
Looks fine! Only one Comment from my side.
The build is currently broken, because the coveralls think is not working correctly: #6
I would merge this pr also without green build, if we fix the build soon.

**Comment by Bogh** [comment]
We are working to fix the travis integration atm. You can merge this one if you like.

**Pull request merged**